Verwerken feedback op tekst bias en non-discriminatie (#147)
MinBZK/Algoritmekader: docs/bouwblokken/fundamentele-rechten/non-discriminatie.md

 
De onderwerpen bias en non-discriminatie spelen daarom een belangrijke rol bij de totstandkoming van verantwoord ontwikkelde algoritmen en AI en het gebruik daarvan door ambtenaren. 

## Wat is discriminatie en bias?

### Discriminatie
[Artikel 1 van de Nederlandse Grondwet](https://wetten.overheid.nl/jci1.3:c:BWBR0001840&hoofdstuk=1&artikel=1&z=2023-02-22&g=2023-02-22) verbiedt discriminatie: 

> Allen die zich in Nederland bevinden, worden in gelijke gevallen gelijk behandeld. Discriminatie wegens godsdienst, levensovertuiging, politieke gezindheid, ras, geslacht, handicap, seksuele gerichtheid of op welke grond dan ook, is niet toegestaan. 

> Non-discriminatie is dus de basis van ons rechtssysteem en onze samenleving. 

Reviewer: Ik vind het ver gaan om het de basis van ons rechtssysteem te noemen. Weet niet of dat uit externe review kwam. Misschien "De prominente positie in de Grondwet benadrukt het belang van het mensenrecht in Nederland"?

Author: Hm ik snap wat je bedoelt. Feedback was om te beginnen bij artikel 1 van de grondwet, eigenlijk net zoals de handreiking non-discriminatie doet. De handreiking verwoort dit ook zo. Iets genuanceerder lijkt me beter!

> Non-discriminatie is dus de basis van ons rechtssysteem en onze samenleving. 

Author: ```suggestion
De prominente positie in de Grondwet benadrukt het belang van het mensenrecht in Nederland. 
```

> De afgelopen jaren is duidelijk geworden dat ook algoritmes en AI-systemen discriminatoire effecten kunnen hebben.  

Reviewer: Misschien: "De afgelopen jaren hebben incidenten in de praktijk de aandacht gericht op de discriminatoire effecten die algoritmes en AI-systemen kunnen hebben". Immers is het op zich al decennia duidelijk dat algoritmes kunnen discrimineren.

> De afgelopen jaren is duidelijk geworden dat ook algoritmes en AI-systemen discriminatoire effecten kunnen hebben.  

Author: ```suggestion
De afgelopen jaren hebben incidenten in de praktijk de aandacht gericht op de discriminatoire effecten die algoritmes en AI-systemen kunnen hebben.
```

Soms worden algoritmes en AI bijvoorbeeld juist ingezet om op een zo objectief mogelijke manier te bepalen welke groepen meer of minder belang hebben bij een andere behandeling. 
In deze gevallen zal er altijd na moeten worden gegaan of er sprake is van een objectieve rechtvaardiging voor het gemaakte onderscheid.

Wanneer er geen rechtvaardiging is voor het gemaakte onderscheid, spreken we van een verboden direct of indirect onderscheid, ofwel discriminatie. 
Het algoritme of AI-systeem mag in dat geval niet gebruikt worden.

> Bias vormt daarmee een risico voor discriminatie.

Reviewer: *op ipv voor?

> Bias vormt daarmee een risico voor discriminatie.

Author: ```suggestion
Bias vormt daarmee een risico op discriminatie.
```

##### Fairness 
In de context van algoritmes en AI wordt de term *unfairness* gebruikt wanneer er sprake is van een ongerechtvaardigd onderscheid waarbij bepaalde groepen meer bevoordeeld worden dan andere.[^5] 
In de Nederlandse taal spreken we dan van oneerlijkheid of onrechtvaardigheid (of in positieve zin van respectievelijk fairness, eerlijkheid en rechtvaardigheid). 
Wanneer we het hebben over fairness hebben we het minder over de juridische kant van discriminatie en verboden onderscheid. 

> Ook als er wel een objectieve rechtvaardiging bestaat voor een gemaakt onderscheid, kan je je afvragen of het gemaakte onderscheid ethisch wenselijk is. 

Reviewer: *kan afgevraagd worden of het...

> Ook als er wel een objectieve rechtvaardiging bestaat voor een gemaakt onderscheid, kan je je afvragen of het gemaakte onderscheid ethisch wenselijk is. 

Author: ```suggestion
Ook als er wel een objectieve rechtvaardiging bestaat voor een gemaakt onderscheid, kan afgevraagd worden of het gemaakte onderscheid ethisch wenselijk is. 
```

is verboden. Daarnaast wordt expliciet vermeld dat binnen de werkingssfeer van de Verdragen en onverminderd de bijzondere bepalingen ervan, iedere discriminatie op grond van nationaliteit verboden is.

## Vereisten

<!-- list_vereisten bouwblok/bias-en-non-discriminatie -->

## Maatregelen

<!-- list_maatregelen bouwblok/bias-en-non-discriminatie -->

> !!! note "Aanbevelingen vanuit documenten (nog te verwerken)"

Reviewer: Dit kan misschien beter weg voor deze release?

> !!! note "Aanbevelingen vanuit documenten (nog te verwerken)"

Author: ```suggestion
```

> 
>     [Rathenau](https://www.rathenau.nl/nl/digitalisering/algoritmes-afwegen)
> 
>     * Geef als uitvoeringsorganisatie meer inzicht in hoe biastoetsing plaatsvindt 
>     * Zet een nationaal kennisplatform voor biastoetsing op waar expertise kan worden ontwikkeld en gedeeld. Bepaal welke mate van standaardisatie gewenst is en of wettelijke eisen nodig zijn.
> 
>     [ADR](https://open.overheid.nl/documenten/7052294a-e70a-4084-88da-d09ae5f202cb/file)
> 
>     * Plaats de handreiking in een kader in relatie tot andere instrumenten 
>     * Overweeg een risicogerichte benadering voor de toepassing van de handreiking 
>     * Werk aan het vergroten van bewustzijn voor algoritmen en (data-)ethiek in de organisatie 
>     * Zorg voor duidelijkheid in taken en verantwoordelijkheden van verschillende betrokkenen 
>     * Beleg verantwoordelijkheid voor de handreiking en borg de (blijvende) aandacht ervoor 
>     * Verplichte toepassing van de handreiking kan bestaande initiatieven tenietdoen 
> 
>     [Toetsingskader ADR](https://open.overheid.nl/documenten/61b54381-d331-40ed-8fce-b2883b195f25/file)
> 
>     * De definitie van de verschillende groepen en de gewenste prestatie van het model voor deze groepen zijn opgenomen in de functionele eisen.
>     * De mate van geaccepteerde bias in de uitkomst is opgenomen in de functionele eisen en uitgewerkt in meetbare prestatiecriteria.
>     * De methoden om bias te voorkomen, detecteren en corrigeren zijn vastgelegd.
>     * De mate van bias in de data, dataverzameling en het model zijn in kaart gebracht.
>     * Tijdens de ontwikkeling van het model is beoordeeld of er een verschil bestaat tussen de prestatie van het model tussen verschillende subgroepen. De prestatiemetrieken afleidbaar uit de confusionmatrix zijn vergeleken voor deze subgroepen.
>     * De uitkomstbias van productiedata is beoordeeld voor de verschillende subgroepen en voldoet aan de prestatiecriteria.
>     * Bij de geconstateerde bias is beoordeeld of deze op discriminatie duidt.
> 
>     [College voor de Rechten van de Mens](https://publicaties.mensenrechten.nl/publicatie/61a734e65d726f72c45f9dce)
> 
>     * Overheidsinstanties mogen bij opsporings- en handhavingsbevoegdheden, met het oog op effectiviteit, efficiëntie en kostenbesparing, gebruik maken van risicoprofielen. Binnen deze risicoprofielen mogen ervaringsgegevens die tot een bepaalde vooronderstelling leiden een rol spelen, tenzij dit leidt tot discriminatie op grond van ras of nationaliteit
>     * Risicoprofielen die uitsluitend of in doorslaggevende mate gebaseerd zijn op ras (waaronder etniciteit en afkomst) zijn in strijd met het discriminatieverbod;
>     * Risicoprofielen die zich richten op één bepaalde afkomst of nationaliteit hebben een stigmatiserend effect en zijn daarom strijdig met het discriminatieverbod;
>     * Risicoprofielen die uitsluitend gebaseerd zijn op nationaliteit zijn zeer moeilijk te rechtvaardigen;
>     * Risicoprofielen waarin ras of nationaliteit mede een rol speelt, kunnen slechts gerechtvaardigd worden door zeer zwaarwegende redenen;
>     * Het gebruik van ras of nationaliteit als selectiecriterium binnen een risicoprofiel is nooit toegestaan als er geen objectieve relatie kan worden aangetoond tussen dit selectiecriterium en het legitieme doel van het profiel;
>     * In alle gevallen moeten de selectiecriteria  binnen een risicoprofiel samen voldoende relevant en objectief (geschikt) zijn om op een effectieve wijze bij te dragen aan de verwezenlijking van het nagestreefde legitieme doel;
>     * Het gebruik van ras of nationaliteit als selectiecriterium binnen een risicoprofiel moet daarnaast noodzakelijk zijn om het gewenste doel tebereiken.
>     * Selectiebeslissingen moeten te allen tijde uitlegbaar zijn.
>     

Author: ```suggestion
```
